src/styles/shared/variables.css

```diff
@@ -1,3 +1,5 @@
 :root {
+	--grid-border-color: rgb(221, 221, 221);
```

Comment:
It's worth noting that the rest of the `dojo2` widgets all use `#123abc' colours as opposed to `rgb`. Also pixels were chosen over `em` in most cases.